kernel/arch/dreamcast/hardware/pvr/pvr_dma.c

```diff
@@ -141,24 +122,24 @@ int pvr_dma_transfer(void * src, uint32 dest, uint32 count, int type,
 }
 
 /* Count is in bytes. */
-int pvr_txr_load_dma(void * src, pvr_ptr_t dest, uint32 count, int block,
+int pvr_txr_load_dma(void * src, pvr_ptr_t dest, uint32_t count, int block,
```

Comment:
```suggestion
int pvr_txr_load_dma(void * src, pvr_ptr_t dest, size_t count, int block,
```

Reply:
Not seeing the corresponding header file change for these couple changes? I might be being dumb, but don't forget to update them.